bokehjs/src/lib/models/axes/categorical_axis.ts

```diff
@@ -24,6 +24,21 @@ export class CategoricalAxisView extends AxisView {
   declare model: CategoricalAxis
   declare visuals: CategoricalAxis.Visuals
 
+  protected  _hit_value(sx: number, sy: number): any | null {
```

Comment:
Here in a few other places:
```suggestion
  protected _hit_value(sx: number, sy: number): any | null {
```

Reply:
[931b309](https://github.com/bokeh/bokeh/pull/14050/commits/931b3099e22f90acc337e5264c9b2bbb48750b9e)